backend/private-graph/graph/schema.resolvers.go

```diff
@@ -77,23 +77,24 @@ import (
 
 // EnableBusinessDashboards is the resolver for the enable_business_dashboards field.
 func (r *allWorkspaceSettingsResolver) EnableBusinessDashboards(ctx context.Context, obj *model.AllWorkspaceSettings) (bool, error) {
-	w, err := r.isUserInWorkspaceReadOnly(ctx, obj.WorkspaceID)
```

Comment:
I think we should keep this check - at the moment this doesn't return anything but a boolean but the function could be changed in the future without realizing there's no permission check